Hypospray Timer

Fre3bie opened the pull request:
Recently there has apparently been an increase in CMO's wanting to get their valids with loaded hypo's of chloral hydrate for that sweet instant knockout. To combat this, a small delay has been added to the hypo injection to give a fighting chance to someone on the receiving end of an unwanted injection while still being a useful tool for the proper purpose of treating patients.

:cl: Fre3bie
tweak: Small delay added to hypospray injection to deter their use as insta-KO weapons.
/:cl:

Spookerton commented on code/modules/reagents/reagent_containers/hypospray.dm:
```diff
@@ -83,7 +83,7 @@
 	amount_per_transfer_from_this = 5
 	volume = 0
-	time = 0 // hyposprays are instant for conscious people
+	time = 7 // Faster than a syringe, but not instant to prevent cheesing people with insta-chloral injections.
```
```suggestion
	time = 7
```

Spookerton reviewed (dismissed):
No need for justification comments.

Fre3bie committed:
Hypospray Timer

Changes hypo from instant to a small delay to prevent validhunting with chloral sprays.
